data/nlu/nlu.md

```diff
@@ -1377,6 +1385,8 @@
 - you are ai
 - you are chatbot
 - you're a bot
+- are u human
+- are you human
```

Comment:
so um i hesitated adding "are you human" to this intent, because the answer to this question is "i am indeed a bot" or similar... maybe we should have a new intent for "are you a human"?

Reply:
Yeah that's a good point. Was wondering if there was a good response that could cover both, but doesn't seem like it. Then could `are you real` go under `are you human`?

Reply:
removed these 2, making a PR for that here: https://github.com/RasaHQ/rasa-demo/pull/509